samples/nodejs-ts/README.md

```diff
@@ -0,0 +1,36 @@
+# NodeJS + Typescript
```

Comment:
I don't know why, but in your PR appeared changes from other. Probably rebasing in 6730388598f8aec243213284b21f517db5248af3 went wrong.

Could I please you to reset code into 77d01f3a5c22c5f9964f05ca7656b9e1da2dbefe ?

```
git reset --hard 77d01f3a5c22c5f9964f05ca7656b9e1da2dbefe
```
and later use `git merge` commend to get changes from upstream master ?